examples/default.yaml

```diff
@@ -5,6 +5,16 @@
 # Default values in this YAML file are specified by `null` instead of Lima's "builtin default" values,
 # so they can be overridden by the $LIMA_HOME/_config/default.yaml mechanism documented at the end of this file.
 
+# A template should specify the minimum Lima version required to parse this template correctly.
+# It should not be set if the minimum version is less than 1.0.0
+# 🟢 Builtin default: not set
+minimumLimaVersion: null
+
+# Minimum version of QEMU required to create an instance of this template.
+# Will be ignored if the vmType is not "qemu"
+# 🟢 Builtin default: not set
+minimumQEMUVersion: null
+
```

Comment:
```suggestion
vmOpts:
  qemu:
    minimumVersion: null
```

Reply:
Feels a bit too deeply nested right now, but I guess we can put additional stuff (something like `QEMU_SYSTEM_X86_64`) under there as well in the future.